### PR TITLE
Fix website deploy with java 17

### PR DESCRIPTION
**Pull request by imsdu**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -11,9 +11,11 @@ addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent"       % "0.1.6")
...
 addSbtPlugin("com.eed3si9n"      % "sbt-buildinfo"       % "0.10.0")
 
-addSbtPlugin("com.typesafe.sbt"      % "sbt-site"                   % "1.4.1")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"                % "0.9.2")
-addSbtPlugin("io.github.jonas"       % "sbt-paradox-material-theme" % "0.5.1")
+addSbtPlugin(
+  ("com.typesafe.sbt"                % "sbt-site"                   % "1.4.1").exclude("org.scala-lang.modules", "scala-xml_2.12")
+)
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"                % "0.10.3")
+addSbtPlugin("io.github.jonas"       % "sbt-paradox-material-theme" % "0.6.0")
```
Comment (olivergrabinski) [inline project/plugins.sbt:18]: Wasn't there some kind of issue with this version?

Comment (imsdu) [reply project/plugins.sbt:18]: Hum, I set the same one as 1.8 but there is indeed a newer one.
I only remember the problem with 0.9.x but I will set the newer version

**Review by shinyhappydan — APPROVED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (olivergrabinski) [reply project/plugins.sbt:18]: I actually meant the question for `sbt-paradox-material-theme`, I thought it was breaking something on the website

Comment (imsdu) [reply project/plugins.sbt:18]: This one has not changed in a loooong time

Comment (olivergrabinski) [reply project/plugins.sbt:18]: Isn't it being updated from `0.5.1` to `0.6.0` in this PR?

Comment (imsdu) [reply project/plugins.sbt:18]: Sorry, I meant the plugin itself is not maintained anymore, I must have misread your question.
Those are both the paradox versions that we use in the main branch

**Pull request merged**